ambry-utils/src/main/java/com.github.ambry.utils/Time.java

```diff
@@ -31,11 +31,6 @@
   public static final int NsPerSec = NsPerMs * MsPerSec;
   public static final int UsPerSec = UsPerMs * MsPerSec;
   public static final int SecsPerMin = 60;
```

Comment:
SecsPerMin is used here https://github.com/linkedin/ambry/blob/master/ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java#L226